From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Thu, 7 Apr 2022 12:56:28 +0300
Subject: [PATCH] Fix build on Qt 5.6

The compose plugin as fallback not available even on sailfish.
---
 input-context/minputcontext.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/input-context/minputcontext.cpp b/input-context/minputcontext.cpp
index 94030936c18e816b06237c789d3f1b4456ee632e..fa0ad63cf7f350108851e469a026c9e9839d4ee7 100644
--- a/input-context/minputcontext.cpp
+++ b/input-context/minputcontext.cpp
@@ -61,8 +61,12 @@ MInputContext::MInputContext()
       preeditCursorPos(-1),
       redirectKeys(false),
       currentFocusAcceptsInput(false),
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 8, 0))
       composeInputContext(qLoadPlugin<QPlatformInputContext, QPlatformInputContextPlugin>
                           (loader(), "compose", QStringList()))
+#else
+      composeInputContext(nullptr)
+#endif
 {
     QByteArray debugEnvVar = qgetenv("MALIIT_DEBUG");
     if (!debugEnvVar.isEmpty() && debugEnvVar != "0") {
